### PR TITLE
Altering content to make form generic for phone and f2f appointments

### PR DESCRIPTION
**Pull request by Guntrisoft**
create summary document 1

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <%= stylesheet_link_tag 'application', media: 'all' %>
 <% end %>
 
-<% content_for :app_title, 'Pension Wise Summary Documents' %>
+<% content_for :app_title, 'Summary document generator' %>
```
Comment (benlovell) [inline app/views/layouts/application.html.erb]: Should this be title-case?

**Comment by Guntrisoft** [comment]
[Output content changes to make it more generic for both face-to-face and telephone appointments](https://trello.com/c/Gp4XGDL8/177-output-content-changes-to-make-it-more-generic-for-both-face-to-face-and-telephone-appointments)

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (Guntrisoft) [inline app/views/layouts/application.html.erb]: I shall refer this to @guidance-guarantee-programme/content-designers

Comment (Guntrisoft) [inline app/views/layouts/application.html.erb]: It does seem to be the style of other apps (titled-cased)

Comment (benlovell) [inline app/views/layouts/application.html.erb]: It's also literally going in the 'title' 😄

Comment (Guntrisoft) [inline app/views/layouts/application.html.erb]: changed

**Pull request merged**